src/issue/muteIssue.ts

```diff
@@ -0,0 +1,99 @@
+/* --------------------------------------------------------------------------------------------
+ * SonarLint for VisualStudio Code
+ * Copyright (C) 2017-2023 SonarSource SA
+ * sonarlint@sonarsource.com
+ * Licensed under the LGPLv3 License. See LICENSE.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+'use strict';
+
+import { QuickPickItem, window } from 'vscode';
+import { MultiStepInput } from '../util/multiStepInput';
+import { IssueService } from './issue';
+
+
+const WONT_FIX_STATUS = 'Won\'t fix';
+const FALSE_POSITIVE_STATUS = 'False positive';
+export const MUTE_TRANSITION_STATES: QuickPickItem[] = [WONT_FIX_STATUS, FALSE_POSITIVE_STATUS]
+  .map(label => ({ label }));
+
+export async function muteIssueMultiStepInput(
+  workspaceUri: string,
+  issueKey: string,
+  fileUri: string,
+  isTaintIssue: boolean) {
+
+  interface State {
+    title: string;
+    step: number;
+    totalSteps: number;
+    transitionState: QuickPickItem | string;
+    comment: string;
+  }
+
+  const title = 'Mute Issue';
+
+  async function muteIssue(input: MultiStepInput, state: Partial<State>) {
+    const pickedIssueStatus = await pickIssueStatus(input, title, MUTE_TRANSITION_STATES);
+    if (pickedIssueStatus) {
+      const comment = await inputComment(input, state);
+      window.showInformationMessage('Do you want to do this?', {
+          modal: true,
+          detail: 'This action will change the status of the issue on the SonarQube/SonarCloud ' +
```

Comment:
Is there a way to get the information on whether the issue comes from SonarQube or SonarCloud, to show the appropriate product name?